packages/fast-form-generator-react/src/form/form-item.children.style.ts

```diff
@@ -0,0 +1,93 @@
+import { toPx } from "@microsoft/fast-jss-utilities";
+import {
+    applyAriaHiddenStyles,
+    applyCleanListStyle,
+    applyInputStyle,
+    applyListItemStyle,
+    applyPopupHeadingStyles,
+    applyPopupMenuStyles
+} from "../utilities/form-input.style";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { IFormItemChildrenClassNameContract } from "../class-name-contracts/";
+
+const styles: ComponentStyles<IFormItemChildrenClassNameContract, {}> = {
+    formItemChildren: {
+        display: "flex",
+        flexDirection: "column"
+    },
+    formItemChildren_inputWrapper: {
+        display: "flex",
+        paddingBottom: toPx(12),
+        "& input": {
+            ...applyInputStyle(),
+            flex: "2",
+            marginRight: toPx(8)
+        },
+        "& button": {
+            fontSize: toPx(15),
+            maxWidth: toPx(374),
+            minWidth: toPx(120),
+            flex: "1",
+            display: "inline-block",
+            padding: `${toPx(13)} ${toPx(12)} ${toPx(12)}`,
+            border: `${toPx(2)} solid transparent`,
+            borderRadius: toPx(2),
+            cursor: "pointer",
+            overflow: "hidden",
+            lineHeight: "1",
+            textAlign: "center",
+            whiteSpace: "nowrap",
+            verticalAlign: "bottom",
+            transition: "all 0.2s ease-in-out",
+            color: "white",
+            background: "#0078D4",
+            "&:hover": {
+                background: "#0075CF"
```

Comment:
can we have a file with exported object with these values? i feel they have been used enough time throughout the styling to warrant it.